packages/cbioportal-utils/src/signal/DisplayUtils.ts

```diff
@@ -0,0 +1,11 @@
+export function fixDisplayValue(value: number | null) {
```

Comment:
Actually, looks like we already have a very similar function here: https://github.com/cBioPortal/cbioportal-frontend/blob/97d211b37627ddc9f46d7044d2f4f653db49576c/packages/cbioportal-utils/src/format/FormatUtils.ts#L1

We should probably reuse that one. In any case this function is not really signal specific, so we should probably move the implementation into `FormatUtils`.

Reply:
Good point! The one we have has one more checking for null value, and if it's null we show `-`. So I move the function to `FormatUtils` and resue the `formatPercentValue`